RestSharp/Http.Sync.cs

```diff
@@ -201,6 +201,9 @@ protected virtual HttpWebRequest ConfigureWebRequest(string method, Uri url)
             webRequest.PreAuthenticate = PreAuthenticate;
             webRequest.Pipelined = Pipelined;
             webRequest.UnsafeAuthenticatedConnectionSharing = UnsafeAuthenticatedConnectionSharing;
+#if NETSTANDARD2_0
+            webRequest.Proxy = null;
```

Comment:
Shouldn't we check if there is a proxy set for the RestClient and use this value? I believe with this change we will not be able to manually set proxy.

Reply:
This is just that the next line setting Expect100Continue does not fail.
The proxy is set again further down [on line 243](https://github.com/MichelZ/RestSharp/blob/23474c3fe4e7803ae79f3affba3f818bd2f584e2/RestSharp/Http.Sync.cs#L243).

We could move it closer to the top so it's obvious that it gets set again?

The included ProxyTest does work setting the proxy manually to something nonexistent and we get the correct Exceptions in both, Full Framework and in .NET Core 2.0.